superchain/extra/addresses/addresses.json

```diff
@@ -209,12 +209,12 @@
     "OptimismPortalProxy": "0x76114bd29dFcC7a9892240D317E6c7C2A281Ffc6",
     "SystemConfigProxy": "0xa6b72407e2dc9EBF84b839B69A24C88929cf20F7",
     "ProxyAdmin": "0x18d890A46A3556e7F36f28C79F6157BC7a59f867",
-    "DelayedWETHProxy": "0x4BC348b055785BC6Bdf8d8b3C72609C683580a06",
-    "DisputeGameFactoryProxy": "0x8c8341e39837Dd44dd2dcA45350FDa228392D558",
-    "Mips": "0xCaAe0e301210B1976eD63478b7cF9A03F4E2134a",
-    "PreimageOracle": "0x3C3B1B2f8FbE2e91411DdC7ad358d1EBe2e44FB5",
-    "FaultDisputeGame": "0x016ae406c9F21F01cA39D05117378F0c523bD8C1",
-    "PermissionedDisputeGame": "0xF795Ad48726B1cD3E7B66a70E5AdA91D6D593132"
+    "DelayedWETHProxy": "0x494E1970923A836Ce00Bdf83dCe4b13981C95039",
+    "DisputeGameFactoryProxy": "0x2419423C72998eb1c6c15A235de2f112f8E38efF",
+    "Mips": "0xCdD4eabeb5f5d1fD195b15AFBF4eC3CA605Ec8Fe",
+    "PreimageOracle": "0xc5BF95A758CA8F0695081d6Ef4F57E204DF2bc4A",
```

Comment:
The correct `PreimageOracle` address is `0xA6662E943C763b23e82eEf3868E50007C9Af10bf` (can confirm this by calling `fdg.vm().oracle()`)
(the address on notion is incorrect)

Reply:
I was pulling the addresses from on-chain:
```
$cast-sepolia call 0x2419423C72998eb1c6c15A235de2f112f8E38efF 'gameImpls(uint32)(address)' 0
0x3CdB0e38bC990c07eADA1376248BB2a405Ae3B9B

cast-sepolia call 0x3CdB0e38bC990c07eADA1376248BB2a405Ae3B9B 'vm()(address)'
0xCdD4eabeb5f5d1fD195b15AFBF4eC3CA605Ec8Fe

cast-sepolia call 0xCdD4eabeb5f5d1fD195b15AFBF4eC3CA605Ec8Fe 'oracle()(address)'
0xc5BF95A758CA8F0695081d6Ef4F57E204DF2bc4A
```

Reply:
Very odd. Could've sworn it's a different address I got. Though I used etherscan rather than my sepolia rpc.
I checked again and have confirmed that the po address in the PR is correct. Although, it's an older version at 0.1.0 which doesn't have the LPP receipt logs. I'll fix that next week.